tests/test_singer.py

```diff
@@ -192,6 +192,14 @@ def test_parse_bulk_decs(self):
             value = self.create_record(value_str)
             self.assertEqual(float(value_str), value)
 
+    def test_format_message(self):
```

Comment:
a new parameter ('option') is added to the method which is not tested here at all. and also this test will be passed with any value for `option`

Reply:
thanks, I added a couple of more tests. The `option` is an optional parameter defaults to `0`, means no extra options. If the option parameter not provided then it should not append newlines.

Is there anything I should add?

Reply:
It looks good to me 🙏